docs/contribute/README.md

```diff
@@ -41,7 +42,7 @@ want to begin.
 
 ### FlowForge Code Structure
 
-```
+```console
```

Comment:
Can't use console here. You break syntax highlighting. Please remove it

Reply:
Odd, works for me! Will remove.